tests/EdFi.Security.DataAccess.UnitTests/Contexts/SecurityContextFactoryTests.cs

```diff
@@ -9,37 +9,36 @@
 using NUnit.Framework;
 using Shouldly;
 using EdFi.Ods.Common.Configuration;
+using EdFi.Ods.Common.Database;
 
 namespace EdFi.Security.DataAccess.UnitTests.Contexts
 {
     [TestFixture]
     public class SecurityContextFactoryTests
     {
-        [Test]
-        public void Given_null_argument_then_constructor_throws_exception()
-        {
-            Should.Throw<ArgumentNullException>(() => new SecurityContextFactory(null));
-        }
-
         [Test]
         public void Given_configured_for_SqlServer_then_create_SqlServerSecurityContext()
         {
-            var configurationProvider = A.Fake<IApiConfigurationProvider>();
-            A.CallTo(() => configurationProvider.DatabaseEngine).Returns(DatabaseEngine.SqlServer);
+            var connectionstringProvider = A.Fake<ISecurityDatabaseConnectionStringProvider>();
 
-            new SecurityContextFactory(configurationProvider)
-                .CreateContext()
+            A.CallTo(() => connectionstringProvider.GetConnectionString()).Returns(
```

Comment:
this is a postgres connection string, please update to use the sql server specific string.. remember tests are documentation